components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -204,40 +206,46 @@ static void set_timestamp_info (const nlohmann::json& metadata_json, TimestampIn
 
 TEST_CASE("get_encoding_type", "[ffi][get_encoding_type]") {
     bool is_four_bytes_encoding;
-
     // Test eight-byte encoding
     vector<int8_t> eight_byte_encoding_vec{EightByteEncodingMagicNumber,
                                            EightByteEncodingMagicNumber + MagicNumberLength};
 
-    IrBuffer eight_byte_ir_buffer(eight_byte_encoding_vec.data(),
-                                  eight_byte_encoding_vec.size());
-    REQUIRE(get_encoding_type(eight_byte_ir_buffer, is_four_bytes_encoding) ==
+    // Test eight-byte encoding
+    BufferReader eight_byte_encoding_buffer{
```

Comment:
These name changes seem unnecessary, especially given that you're changing names in the opposite direction (encoding -> ir) in other places.